lxd/container_lxc.go

```diff
@@ -1054,9 +1054,11 @@ func (c *containerLXC) initLXC(config bool) error {
 	}
 
 	// Setup devlxd
-	err = lxcSetConfigItem(cc, "lxc.mount.entry", fmt.Sprintf("%s dev/lxd none bind,create=dir 0 0", shared.VarPath("devlxd")))
-	if err != nil {
-		return err
+	if c.EnableDevLxd() {
```

Comment:
I'd just make that:
```go
if c.expandedConfig["security.devlxd"] == "" || shared.IsTrue(c.expandedConfig["security.devlxd"] {
}
```

And drop the EnableDevLxd() function as I don't really see a use for a public function for this.